hrrecruitment/CRM/HRRecruitment/Upgrader.php

```diff
@@ -49,4 +49,19 @@ public function upgrade_1402() {
 
     return TRUE;
   }
+
+    /**
+     * Upgrade CustomGroup, setting Application, application_case and Evaluation_fields names to is_reserved Yes
+     * @return bool
+     */
+  public function upgrade_1403() {
+      $result = civicrm_api3('CustomGroup', 'get', array(
+          'sequential' => 1,
+          'return' => array("id"),
+          'name' => array('IN' => array("Application", "application_case", "Evaluation_fields")),
```

Comment:
I know the API explorer always uses the `array()` syntax, but for CiviHR we prefer the `[]` syntax. Also, single quotes